src/serlo-editor/core/sub-document/editor.tsx

```diff
@@ -28,16 +28,84 @@ import { usePlugin } from '../contexts/plugins-context'
 import { DocumentEditor } from '@/serlo-editor/editor-ui/document-editor'
 import { EditorPlugin } from '@/serlo-editor/types/internal__plugin'
 
-const hotKeysKeyMap = {
-  FOCUS_PREVIOUS: 'up',
-  FOCUS_NEXT: 'down',
-  INSERT_DEFAULT_PLUGIN: 'enter',
-  DELETE_EMPTY: ['backspace', 'del'],
-  UNDO: ['ctrl+z', 'command+z'],
-  REDO: ['ctrl+y', 'command+y', 'ctrl+shift+z', 'command+shift+z'],
-}
-type HotKeysHandlers = {
-  [K in keyof typeof hotKeysKeyMap]: (keyEvent?: KeyboardEvent) => void
+const useEnableEditorHotKeys = ({
```

Comment:
I'm curious to know if there is any reason for not moving this hook into its own file? There, it could also have its arguments cleanly defined as an interface and have its own import scope.